cmd/syft/internal/options/source.go

```diff
@@ -32,6 +34,7 @@ var _ interface {
 var _ clio.PostLoader = (*imageSource)(nil)
 
 func (o *sourceConfig) DescribeFields(descriptions clio.FieldDescriptionSet) {
+	descriptions.Add(&o.Supplier, ``)
```

Comment:
You should be able to just remove this entire line. It seems to work this way, too, but not necessary to add this when you have a flag defined 👍

Reply:
Removed this entire Add descriptions section